scripts/bundleSize.js

```diff
@@ -0,0 +1,173 @@
+#! /usr/bin/env node
+
+/* eslint-disable no-console */
+
+const ora = require('ora');
+const { execSync } = require('child_process');
+const chalk = require('chalk');
+
+// Size limit for all bundles used by each service (K)
+// Keep these +/- 5K and update frequently!
+const MIN = 605;
+const MAX = 625;
+
+const services = [
```

Comment:
This can be `Object.keys(services)` imported from `https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/loadableConfig.js` :)

Reply:
ah perfect cheers! I tried getting this array by running `readdirSync` on `src/app/lib/config/services` but spotted it had a few extra items I didn't want (like `default` & `index`)

Reply:
@dr3 `src/app/lib/config/services` [needs to use es6 module syntax](https://github.com/jamiebuilds/react-loadable#automatic-code-splitting-on-import) so I think the only way I can import this `services` object is by making `babel-node` a dev dependency - but that seems a bit much..?

Alternatively I could:
- put this services array into a `require`able constants file so it might be reused?
- do as I described above and operate on the array returned by `fs. readdirSync('./src/app/lib/config/services')` to get what's needed

have pushed up the babel-node option but happy to change. Have you got any other ideas?

Reply:
Hmmm tricky one, want to avoid lots of lists of the services, and awkward 'read dir but strip out `default` `index` and `loadableConfig`, but also avoid extra deps :P Kinda a lose lose 🙈 

I think for now, go with `npx @babel/node scripts/bundleSize` and remove the dependancy from the package.json, it matches [this command](https://github.com/bbc/simorgh/blob/latest/package.json#L37). Feel free to raise an issue to refactor this in future, giving a `cjs` way of receiving all services, but for this PR i think using `npx` is fine :)

Reply:
just tried this but `npx` isn't working with `@babel/node` despite the [docs saying it should](https://babeljs.io/docs/en/babel-node)

I can see that I'm not the only person having this problem: https://github.com/babel/babel/issues/8357 - sounds like it might be because `@babel/node`'s executable is named `babel-node`

What are the benefits to using `npx` over having `babel-node` as a dev dependency? Won't more or less all devs and automated jobs still end up downloading the package? Doesn't it also mean it will bypass `snyk` / `dependency-check`?

Reply:
ahh just spotted this on the prod CI job
```
sh: 1: babel-node: not found
```
so leaving @babel/node as a dev dependency isn't an option

Reply:
@dr3 - sorry finally got round to updating this. Am now getting the list of services by running `readdirSync` on `build/public/static/js` and ignoring the filenames that start with `main` / `vendor`. Not the cleanest solution but avoids making `babel-node` a dependency - what do you reckon?

Reply:
Hmm yeah it does solve the problem of needing `babel-node`, my only reservation would be that if the files stop being generated (if bundle splitting breaks) it wont report an error as it only checks the services it can find in bundles. 

But.. you could argue thats not the purpose of this script, and it simply monitors the sizes of what exists :P I think this solution is ok, at least until an `es5` way of getting these service name appears :)

Reply:
ah good point - have added a check for the number of service bundle files :
<img width="931" alt="Screenshot 2019-07-27 at 21 41 50" src="https://user-images.githubusercontent.com/22368483/61999375-80ec8a00-b0b7-11e9-9aba-ebc713a81992.png">